package.json

```diff
@@ -35,6 +35,7 @@
     "chai": "^1.9.1",
     "express": "4.x",
     "grunt": "^0.4.4",
+    "grunt-cli": "^0.1.13",
```

Comment:
I think grunt separated the cli commands from the main base project a while ago. In any case, it's good practice to have everything in the devDependancy that is required to run the tests and locally on my machine I don't have grunt-cli tools installed globally.